src/store/middleware/board.tsx

```diff
@@ -39,6 +39,7 @@ export const passBoardMiddleware = (stateAPI: MiddlewareAPI<Dispatch, Applicatio
 
         if (message.type === "BOARD_DELETED") {
           store.dispatch(Actions.leaveBoard());
+          window.location.assign("/?boardDeleted=true");
```

Comment:
Why don't you dispatch the board deleted toast here?

Reply:
Because the toast disappears as soon as the redirect is finished